tools/vuln_valid/vulnValidate.js

```diff
@@ -31,7 +31,11 @@ const coreModel = joi.object().keys({
   type: joi.string().optional(),
   cvss_score: joi.number().optional(),
   cvss: joi.string().optional(),
-  reported_by: joi.string().optional()
+  reported_by: joi.string().optional(),
+  affectedEnvironments: joi
+  .array()
+  .items(joi.string().valid("mac", "win", "linux"))
```

Comment:
Same as https://github.com/nodejs/security-wg/pull/912#discussion_r1140255261

Reply:
yes, I created a branch from the #912 in order to do the validation before pushing the changes

Reply:
#912 should be merged before this one